www/js/diary/list.js

```diff
@@ -214,12 +215,19 @@ angular.module('emission.main.diary.list',['ui-leaflet',
     }
 
     $scope.populateBasicClasses = function(tripgj) {
-        tripgj.display_start_time = DiaryHelper.getLocalTimeString(tripgj.data.properties.start_local_dt);
-        tripgj.display_end_time = DiaryHelper.getLocalTimeString(tripgj.data.properties.end_local_dt);
+        const start_dt = Object.assign({}, tripgj.data.properties.start_local_dt);
+        const end_dt = Object.assign({}, tripgj.data.properties.end_local_dt);
+        tripgj.isDraft = $scope.isDraft(tripgj);
+        // Fix "Invalid date" for processed trips
+        if (!tripgj.isDraft) {
+          start_dt.month = start_dt.month - 1;
```

Comment:
> tripgj.data.properties.start_local_dt and tripgj.data.properties.end_local_dt use a 1-12 month value while moment needs 0-11 month value. The test was done on December which translated to value 12. Hence moment throw Invalid date

Are you sure this is the problem? Because if this were the case, then the month would always be off by one (e.g. January would be 1 in the `local_dt`, converted to February by moment and displayed as such in the UI), and somebody would have noticed before this. Note that `getLocalTimeString` is used in the master branch as well.

Based on the screenshots, I believe that this fixes the immediate issue that you found, but I am concerned that if it is not based on a well-formed understanding of the underlying bug, it may lead to other regressions.

I would like to see screenshots showing that:
- without this fix, November dates are displayed as December and January dates are displayed as February
- on one of the standard channels (e.g. https://e-mission.eecs.berkeley.edu/#/client_setup?new_client=urap2017emotion&clear_usercache=true&clear_local_storage=true), December dates are not displayed correctly 

before I approve the change.